MdeModulePkg/Bus/Pci/NvmExpressDxe/NvmExpressHci.c

```diff
@@ -730,13 +730,34 @@ NvmeControllerInit (
   NVME_AQA             Aqa;
   NVME_ASQ             Asq;
   NVME_ACQ             Acq;
+  UINT16               VidDid[2];
   UINT8                Sn[21];
   UINT8                Mn[41];
 
+  PciIo = Private->PciIo;
+
+  //
+  // Verify the controller is still accessible
+  //
+  Status = PciIo->Pci.Read (

```

Comment:
Looks unrelated to new features/PCD.  Should it be split into a different commit?

Reply:
Returned old commits, unsquashed.